code/style.css

```diff
@@ -1,4 +1,199 @@
+/* Universal selector: it applies to all of the elements*/
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 html {
-  background: #0025ff;
-  color: #fff;
+  background-color: orange;
+}
+
+body {
+  background-color: orange;
+  min-height: 100vh;
+  min-width: 500px; /*To make the body not to shrink less than 500px*/
+}
+/* HEADER STYLING */
+header {
+  background-color: whitesmoke;
+  height: 10em;
+  width: 100%;
+  position: fixed;
+  top: 0;
+}
+
+nav {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  font-size: 1.3rem;
+}
+
+a {
+  padding: 1rem;
+  text-decoration: none;
+  color: orange;
+}
+
+a:hover {
+  transform: scale(1.2, 1.2);
+  text-decoration: underline;
+}
+
+.logo {
+  width: 70px;
+  border: 1px solid blue;
+  border-radius: 40%;
+  position: absolute;
+  top: 25px;
+  left: 20px;
+}
+
+h1 {
+  text-align: center;
+  margin-top: 1rem;
+  text-decoration: underline;
+  color: blue;
+  font-family: "Newsreader", serif;
+}
+
+/* BIG NEWS SECTION */
+
+section.big-news {
+  margin: 0 auto;
+  margin-top: 11rem;
+  padding: 1.1rem;
+  width: 90%;
+  background-color: lightcyan;
+  border-radius: 50px;
+}
+
+.cat-foto-wrapper {
+  margin: 20px;
+}
+
+.cat-foto-wrapper img {
+  width: 100%;
+  height: auto;
+  border-radius: 40px;
+}
+.cat-foto-wrapper img:hover {
+  transform: scale(1.1, 1.1);
+}
+
+section.big-news h2 {
```

Comment:
wouldn't it be better to use class for h2 for styling?

Reply:
Actually yes, my code looks more complicated!